frontera/contrib/scrapy/converters.py

```diff
@@ -98,8 +99,8 @@ def from_frontier(self, response):
 
 
 def _find_method(obj, func):
-    if obj and hasattr(func, 'im_self') and func.im_self is obj:
-        return func.im_func.__name__
+    if obj and hasattr(func, '__self__') and func.__self__ is obj:
```

Comment:
According to the second to last bullet point under this heading - https://docs.python.org/dev/whatsnew/2.6.html#other-language-changes `__self__` was added as an alternative name in PY2.6. So this should work for PY3 and >=2.6.